openid-connect-4-credential-issuance-1_0.md

```diff
@@ -347,15 +436,71 @@ Below is a non-normative example of a `claims` parameter including the `credenti
 }
 ```
 
+### Pushed Authorization Request
+
+This step is OPTIONAL. In case of complex flows or when the size of the request is large due to the query language or encrypting, clients are RECOMMENDED to use pushed uthorization requests.
+
+#### Example of an Pushed Authorization Request with a static approach of presenting input credentials
+
+Below is a non-normative example of a pushed authorization request:
+
+```
+POST /op/par HTTP/1.1
+    Host: as.example.com
+    Content-Type: application/x-www-form-urlencoded
+
+    &response_type=code
+    &client_id=CLIENT1234
+    &nonce=duk681S8n00GsJpe7n9boxdzen
+    &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb
+    &scope=openid
+    &claims=...
+    &vp_token=...
+```
+
+Below is a non-normative example of a `claims` parameter with `presentation_submission`:
+
 ```json=
-GET /authorize?
-    response_type=code
-    &scope=openid%20https://did.exmaple.org/mDL
-    &client_id=s6BhdRkqt3
-    &state=af0ifjsldkj
-    &redirect_uri=https%3A%2F%2Fclient.example.org%2Fcb HTTP/1.1
+{
+   "credentials":[
+      {
+         "manifest_id":"WA-DL-CLASS-A",
+         "format":"ldp_vc",
+         "presentation_submission":{
+            "id":"a30e3b91-fb77-4d22-95fa-871689c322e2",
+            "definition_id":"32f54163-7166-48f1-93d8-ff217bdb0653",
+            "descriptor_map":[
+               {
+                  "id":"input_1",
+                  "format":"jwt_vc",
+                  "path":"$.verifiableCredential[0]"
+               }
+            ]
+         }
+      }
+   ]
+}
 ```
 
+#### Pushed Authorization Response
+
+The Pushed Authorization Response follows the definition in [@!RFC9126].
+
+### Dynamic Credential Request
+
+This step is OPTIONAL. After receiving an Authorization Request from the Client, the Issuer MAY use this step to obtain additional credentials from the End-user. The Issuer MUST utilize [@OIDC4VP] and [@SIOPv2] to dynamically request additional credentials.
+
+This provides the benefit of the Issuer being able to adhere to the principle of data minimization, for example by including only minimum requirements in the Credentiam Manifest knowing that it can supplement additional information if needed.
+
+In order to enable dynamic callbacks of the issuer to the end-user's wallet, the wallet MAY provide the following additional parameters in the authorization request: 
+
+* `wallet_issuer`: OPTIONAL. JSON String containing the wallet's OpenID Connect Issuer URL. The issuer will use the discovery process as defined in [@SIOPv2] to determine the wallet's capabilities and endpoints. 
```

Comment:
I would aim for a single place with a description of all authz request parameters. Please move those parameters up to the authz parameter section. The section itself can stay as additional explanation of the concept. I suggest to add a forward reference to the parameter description.